.github/ISSUE_TEMPLATE.md

```diff
@@ -1,37 +1,23 @@
 <!--- Remove sections that do not apply -->
 
-This issue tracker is only for technical issues related to bitcoin-core.
+This issue tracker is only for technical issues related to Bitcoin Core.
```

Comment:
All of these notes can be inside a comment block:
```
<!---
Remove sections that do not apply.

This issue tracker is only ...

General bitcoin questions ...

...
--->
```
So that the user can see it when writing but it won't show.

Reply:
uh, I guess it's what @laanwj suggests above.